sys-process/cronbase/cronbase-0.3.7-r10.ebuild

```diff
@@ -25,3 +25,12 @@ src_install() {
 
 	keepdir /var/spool/cron/lastrun
 }
+
+pkg_postinst() {
+	einfo "To add a user to the cron group so it can create cron jobs, run:"
+	einfo
+	einfo "    usermod -a -G cron <user>"
+	einfo
+	einfo "For more information, visit the wiki page:                      "
+	einfo "https://wiki.gentoo.org/wiki/Cron                               "
```

Comment:
Please change it to elog instead of einfo. Also get rid of the trailing spaces on lines 34 and 35.